### PR TITLE
OT-MigrateFlutterEndpoints | Migrate flutter to be hitting our newly created microservices

### LINKED ISSUE #47: OT-MigrateFlutterEndpoints
WHAT TO DO: 
1. Change the WebSocket endpoint to direct to the WebSocket created in the [SessionHandler service](https://github.com/Off-Top-App/SessionHandlerService/blob/master/src/main/java/offTop/SessionHandlerService/Config/WebsocketConfig.java)
2. Make sure when you login and create an account you are using the endpoints from the [User microservice](https://github.com/Off-Top-App/UserService/blob/master/src/main/java/offtop/UserService/Controller/UserController.java)

DONE:
1. You are no longer running **off-top-java** and can login, and successfully send data through the websocket.

### PR DESCRIPTION
**Pull request by frlzjosh**
**WHAT I DID:**
1. Migrated flutter to be hitting our newly created microservices. 
2. Also updated the User object so that it passes the properly named attribute of `profession` instead of `professional`
3.  No longer passing `password` to the backend anymore as it is not required in the User microservice and no longer exists as a column in the user MySQL table

HOW TO TEST:
1. Run Kafka/Zookeeper, [UserService](https://github.com/Off-Top-App/UserService) on master branch, & [SessionHandler](https://github.com/Off-Top-App/SessionHandlerService/tree/OT-SessionHandler). For SessionHandler make sure you're on branch OT-SessionHandler. For this branch, make sure you're on branch OT-MigrateFlutterEndpoints
2. Make sure that you can create an account and can login to the application. Because you are now hitting endpoints from a different microservice when dealing with account creation and login.
3. Make sure that you can send audio data to the WebSocket which now lives in SessionHandlerService microservice
4. Make sure you can Producer.py in [off-top-python](https://github.com/Off-Top-App/off-top-python)/backend-services/Services/Kafka/Producer.py
    - You will need to change the user_id on line 23 to user_id value associated within your MySQL DB of offTopUser.

### CODE REVIEW

File: lib/components/recordingSession/websocket.dart [hunk 1]
```diff
@@ -18,19 +18,21 @@ class MyWebSocket {
   WebSocketChannel channel;
 
   Future<void> sendAudioFile(
-      String exportedAudioData, int userID, String topic) async {
+      String exportedAudioData, int userID, String topic, String time_exported) async {
```
Comment (tonynguyen98) [inline lib/components/recordingSession/websocket.dart]: Can you make `time_exported` to camel case `timeExported`

File: lib/components/recordingSession/websocket.dart [hunk 2]
```diff
@@ -18,19 +18,21 @@ class MyWebSocket {
...
     final List<int> audioData = await processAudioFile(exportedAudioData);
     channel.sink.add(
       json.encode(
-        {'audio_data': audioData, 'user_id': userID.toInt(), 'topic': topic},
+        {'audio_data': audioData, 'user_id': userID.toInt(), 'topic': topic, 'time_exported': time_exported},
```
Comment (tonynguyen98) [inline lib/components/recordingSession/websocket.dart]: change to `'time_exported': timeExported`

**Review by tonynguyen98 — CHANGES_REQUESTED** [review]
Great work, just one single change needed

**Review by tonynguyen98 — APPROVED** [review]
Great work, lots of great updates

File: lib/services/userService.dart [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      createdAt,
+      deletedAt,
+      email,
+      firstName,
+      gender,
+      lastName,
+      profession,
+      userName
+    );
+  }
```
Comment (txt020) [inline lib/services/userService.dart:34]: Looks Good!

File: lib/accountCreation.dart [hunk 4]
```diff
@@ -13,84 +14,59 @@ class SignUp extends StatefulWidget {
...
-      debugPrint(checkCode.toString());
-      throw Exception('Response failed to load');
-    }
-
+    final String createdAt = now.toString();
+    final User userObject = userService.setUserData(
+      ageController.text,
+      cityController.text,
+      createdAt,
+      null,
```
Comment (txt020) [inline lib/accountCreation.dart:52]: Why is there a Null here?

**Review by txt020 — APPROVED** [review]

File: lib/accountCreation.dart (hunk 4 above)
Comment (frlzjosh) [reply lib/accountCreation.dart:52]: because we don't have the time a user's account was deleted at.

**Pull request merged**